lib/ActionSheetCustom.js

```diff
@@ -68,14 +68,16 @@ class ActionSheet extends React.Component {
     Animated.timing(this.state.sheetAnim, {
       toValue: 0,
       duration: 250,
-      easing: Easing.out(Easing.ease)
+      easing: Easing.out(Easing.ease),
+      useNativeDriver: this.props.useNativeDriver || true
     }).start()
   }
 
   _hideSheet (callback) {
     Animated.timing(this.state.sheetAnim, {
       toValue: this.translateY,
-      duration: 200
+      duration: 200,
+      useNativeDriver: this.props.useNativeDriver || true
```

Comment:
suggest change to `this.props.useNativeDriver === true`

Reply:
+1